packages/web-react/src/components/UNSTABLE_ProductLogo/demo/ProductLogoInverted.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import UNSTABLE_ProductLogo from '../UNSTABLE_ProductLogo';
+
+const ProductLogoDefault = () => (
+  <UNSTABLE_ProductLogo isInverted aria-label="Logo of the JobBoard">
+    <svg width="178" height="44" viewBox="0 0 178 44" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
To the future (I'm aware that this is probably a temporary product logo): Isn't possible to separate this code to a component to achieve shareability, readability and to reduce duplicated code?

Reply:
Refactored. SVGs are used as constants